modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.module

```diff
@@ -329,6 +330,66 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_timeline(array &$variables)
   $variables['hide_from'] = $paragraph->get('field_oe_timeline_expand')->value;
 }
 
+/**
+ * Implements hook_preprocess_paragraph() for paragraph--oe-content-row--variant-inpage-navigation.html.twig.
+ */
+function oe_bootstrap_theme_preprocess_paragraph__oe_content_row__variant_inpage_navigation(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+
+  if (!$paragraph->get('field_oe_title')->isEmpty()) {
+    $variables['title'] = $paragraph->get('field_oe_title')->value ?? '';
+  }
+
+  $variables['full_layout'] = TRUE;
+  $variables['scrollspy_id'] = 'bcl-inpage-navigation';
```

Comment:
Concatenate with a random number/string to avoid problems if more than one scrollspy are displayed at the same page.

Reply:
Not possible until delivery of BCL Patterns 0.16.0 where Front team delivers the fix of the OEL-875. Now only one inpage navigation can be created and must have that id to include top and background-color.